### PR TITLE
[Feature-7804][Task] DingTalk alert plugin adds signature.

### LINKED ISSUE #7804: [Feature][Task] DingTalk alert plugin adds signature.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

DingTalk alert plugin adds signature.

### Use case

DingTalk alert plugin adds signature.

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by zhuangchong**
## Purpose of the pull request

this pr close #7804 

image
image

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

File: dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-dingtalk/src/main/java/org/apache/dolphinscheduler/plugin/alert/dingtalk/DingTalkParamsConstants.java [hunk 1]
```diff
@@ -27,6 +27,9 @@
     static final String DING_TALK_KEYWORD = "$t('keyword')";
     static final String NAME_DING_TALK_KEYWORD = "Keyword";
 
+    static final String DING_TALK_SECRET = "$t('secret')";
+    static final String NAME_DING_TALK_SECRET = "secret";
```
Comment (caishunfeng) [inline dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-dingtalk/src/main/java/org/apache/dolphinscheduler/plugin/alert/dingtalk/DingTalkParamsConstants.java]:
```suggestion
    static final String NAME_DING_TALK_SECRET = "Secret";
```

File: dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-dingtalk/src/main/java/org/apache/dolphinscheduler/plugin/alert/dingtalk/DingTalkSender.java [hunk 2]
```diff
@@ -145,8 +159,18 @@ public AlertResult sendDingTalkMsg(String title, String content) {
...
+        StringBuilder text = new StringBuilder();
+        if (org.apache.dolphinscheduler.spi.utils.StringUtils.isNotBlank(keyword)) {
+            text.append(keyword);
+            text.append(":");
+        }
+        text.append(title);
+        text.append("\n");
+        text.append(content);
+
+        String msgToJson = textToJsonString(text.toString());
```
Comment (caishunfeng) [inline dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-dingtalk/src/main/java/org/apache/dolphinscheduler/plugin/alert/dingtalk/DingTalkSender.java:171]: Does the order make a difference?

Comment (zhuangchong) [reply dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-dingtalk/src/main/java/org/apache/dolphinscheduler/plugin/alert/dingtalk/DingTalkSender.java:171]: No effect, the keyword is configured by DingTalk robot, as long as the text message contains it.

**Review by caishunfeng — APPROVED** [review]
LGTM

**Comment by caishunfeng** [comment]
relative issue: #7804

**Pull request merged**